pkg/agent/controller/networkpolicy/reject_test.go

```diff
@@ -502,3 +522,180 @@ func Test_getRejectPacketOutMutateFunc(t *testing.T) {
 		})
 	}
 }
+
+func Test_handleRejectRequest(t *testing.T) {
+	fakeSrcMac := "aa:aa:aa:aa:aa:aa"
+	fakeDstMac := "bb:bb:bb:bb:bb:bb"
+	fakeSrcIPv4 := "1.1.1.1"
+	fakeDstIPv4 := "2.2.2.2"
+	fakeSrcIPv6 := "2001::1"
+	fakeDstIPv6 := "2001::2"
+	fakeSrcPort := uint16(8080)
+	fakeDstPort := uint16(80)
+	fakeTCPSeqNum := uint32(1)
+	fakeOFPort1 := int32(1)
+	fakeOFPort2 := int32(2)
+	fakeSIface := &interfacestore.InterfaceConfig{
+		OVSPortConfig: &interfacestore.OVSPortConfig{
+			OFPort: fakeOFPort2,
+		},
+		MAC: net.HardwareAddr([]byte{0xbb, 0xbb, 0xbb, 0xbb, 0xbb, 0xbb}),
+	}
+	fakeDIface := &interfacestore.InterfaceConfig{
+		OVSPortConfig: &interfacestore.OVSPortConfig{
+			OFPort: fakeOFPort1,
+		},
+		MAC: net.HardwareAddr([]byte{0xaa, 0xaa, 0xaa, 0xaa, 0xaa, 0xaa}),
+	}
+	fakeGWIface := &interfacestore.InterfaceConfig{
+		OVSPortConfig: &interfacestore.OVSPortConfig{
+			OFPort: fakeOFPort1,
+		},
+		MAC: net.HardwareAddr([]byte{0xaa, 0xbb, 0xcc, 0xdd, 0xee, 0xff}),
+	}
+	// Only test RejectPodLocal and RejectServiceRemoteToExternal two reject types,
+	// considering other types branches are covered by the UTs above.
+	tests := []struct {
+		name              string
+		packetIn          *ofctrl.PacketIn
+		antreaProxyEnable bool
+		expectFunc        func(mockClient *oftesting.MockClient, mockIStore *storetesting.MockInterfaceStore)
+	}{
+		{
+			name:              "IPv4,TCP,antreaProxy,podLocal",
```

Comment:
Can we provide a similar name like those in `Test_getRejectPacketOutMutateFunc`? eg: "RejectPodLocalWithAntreaProxy..."? I feel it might be easier to understand the case.

Reply:
Done.